portal/templates/portal_wrapper.html

```diff
@@ -59,7 +59,12 @@
                   .on("click", "#log-out", sessMon.logout)
                   .find("#remaining-time").text(sessMon.timeBeforeWarning / 1000);
        {% endif %}
-     });
+
+       (function() {
+        var __env = {% if config.SYSTEM_TYPE %}"{{config.SYSTEM_TYPE}}"{% else %}""{% endif %};
+        if ((__env != "" && __env.toLowerCase() != "production") && ($(".watermark").length == 0)) $("<div class='watermark'>TRUE<sup>NTH</sup> - " + __env + " version - Not for study or clinical use</div>").appendTo("body");
```

Comment:
Could you add a default of "development" to [config.py](https://github.com/uwcirg/true_nth_usa_portal/blob/develop/portal/config.py#L89-L92) as well? We'll need to override it on every production system (just us.truenth.org for now) in application.cfg (I'll take care of that as soon as this is merged)